### DISCUSSION TITLE
hide toctree content from left sidebar

### DISCUSSION THREAD

**Discussion by mluerig** [pradyunsg/furo#328] [Ideas] [answered]
On my starting page I would like to have detailed tocs in the body (down to second level) without having them replicated on the sidebar. In other words: can we have tocs and "hide" them from the left sidebar?

**Comment by pradyunsg** [comment, marked as answer]
> can we have tocs and "hide" them from the left sidebar?

Assuming that you're referring to `toctree` directives... no, that's not possible with Furo's sidebar design.
